src/numbuf/python/src/pynumbuf/numbuf.cc

```diff
@@ -7,16 +7,9 @@
 
 #include <iostream>
 
-#include <arrow/api.h>
-#include <arrow/ipc/api.h>
-#include <arrow/ipc/writer.h>
-
-#include <arrow/python/numpy_convert.h>
-
-#include "adapters/python.h"
-#include "memory.h"
-
 #ifdef HAS_PLASMA
+#include "plasma_common.h"
+
```

Comment:
If the space is not there, clang-format tries to reorder the includes to sort them alphabetically, but this include needs to come first, so I think we do need the space here.